src/ui/Card/Card.tsx

```diff
@@ -30,6 +30,7 @@ Header.displayName = 'Card.Header'
 const title = cva(['font-semibold'], {
   variants: {
     size: {
+      sm: ['text-sm'],
```

Comment:
Sorta sad this wasn't added beforehand =P

Reply:
One would think a header should not be small, but alas the non-Card cards on this page use small text. I did make a ticket to update all of these to Card though.

Reply:
🫡